packages/pluggableWidgets/slider-web/src/Slider.tsx

```diff
@@ -66,9 +68,10 @@ export default function Slider(props: SliderContainerProps): ReactNode {
         maxAttribute,
         expressionMaximumValue
     });
-    const handle = createHandleGenerator({ tooltip, showTooltip, tooltipType });
+    const handle = createHandleGenerator({ tooltip, showTooltip, tooltipType, tooltipAlwaysVisible });
     const { onChange } = useOnChangeDebounced({ valueAttribute, onChange: onChangeProp });
     const style = getStyleProp({ orientation, height, heightUnit });
+    useScheduleUpdateOnce(() => isAvailable(valueAttribute));
```

Comment:
I don't quite get why this is necessary, can you explain what we're trying to achieve?  What flow does it fix?

From what I understood: you want to force this component to re-render when the availability changes from `valueAttribute`? But that would already be happening when the `valueAttribute` changes due to its `status` prop changing. So I'm a bit confused at why we need this.

Reply:
[Answered](https://github.com/mendix/widgets-resources/pull/1151/files/6811defcb4e784f4540c87d719abd455552781a6#r773009625) this question above

Reply:
@Keraito, fixed